client/app/screens/Loading.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Button, Image, Linking } from 'react-native';
+import Login from 'react-native-login';
+
+import {
+  Body,
+  Card,
+  CardItem,
+  Col,
+  Container,
+  Content,
+  Grid,
+  H3,
+  Icon,
+  ListItem,
+  Row,
+  Text,
+  variables,
+  View,
+} from 'native-base';
+
+const config = {
```

Comment:
Can we call this something like `authConfig` here and in `Login.js`? `config` is a little vague.